package.json

```diff
@@ -63,6 +63,6 @@
   },
   "resolutions": {
     "@types/react": "18.2.74",
-    "xml2js": "0.5.0"
+    "xml2js": "0.6.2"
```

Comment:
@Zangetsu101 do you happen to remember why we have this version pinned down? Seems its a transient dependency of minio

Reply:
https://github.com/opencrvs/opencrvs-core/pull/5098/commits/ab2f478698951a8fd9b0c9030e3b402c05fabcbb

Minio at the time was using a pinned version of this package which had some vulnerabilities. It should be safe to remove this now